components/brave_rewards/resources/internals/components/externalWallet.tsx

```diff
@@ -0,0 +1,57 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import * as React from 'react'
+
+// Utils
+import { getLocale } from '../../../../common/locale'
+
+interface Props {
+  info: RewardsInternals.ExternalWallet
+}
+
+const getWalletStatus = (status: RewardsInternals.WalletStatus) => {
+  switch (status) {
+    case 0: {
+      return getLocale('walletStatusNotConnected')
+    }
+    case 1: {
+      return getLocale('walletStatusConnected')
+    }
+    case 2: {
+      return getLocale('walletStatusVerified')
+    }
+    case 3: {
+      return getLocale('walletStatusDisconnectedNotVerified')
+    }
+    case 4: {
+      return getLocale('walletStatusDisconnectedVerified')
+    }
+    case 5: {
+      return getLocale('walletStatusPending')
+    }
+  }
+
+  return 'No wallet'
```

Comment:
This string doesn't require localization?

Reply:
no, this is just something to print out if we don't have handler for it. Should never be displayed in the UI